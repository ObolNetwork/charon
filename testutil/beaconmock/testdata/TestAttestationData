{
 "slot": "1",
 "index": "2",
 "beacon_block_root": "0xfdfe13ba636597b8159043eb944a7c3f00000000000000000000000000000000",
 "source": {
  "epoch": "81327",
  "root": "0xcfda9f2b65659729f7db809a8734466800000000000000000000000000000000"
 },
 "target": {
  "epoch": "81328",
  "root": "0xfdfe13ba636597b8159043eb944a7c3f00000000000000000000000000000000"
 }
}