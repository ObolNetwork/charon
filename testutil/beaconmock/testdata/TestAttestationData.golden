{
 "slot": "1",
 "index": "2",
 "beacon_block_root": "0xa8c4c6a0a5659b7dda195037680a0b6200000000000000000000000000000000",
 "source": {
  "epoch": "8567",
  "root": "0xb5705da8db659b993a3cdea7e15ccead00000000000000000000000000000000"
 },
 "target": {
  "epoch": "8568",
  "root": "0xa8c4c6a0a5659b7dda195037680a0b6200000000000000000000000000000000"
 }
}